lottie-compose/src/main/java/com/airbnb/lottie/compose/animateLottieComposition.kt

```diff
@@ -0,0 +1,117 @@
+package com.airbnb.lottie.compose
+
+import androidx.compose.animation.core.*
+import androidx.compose.runtime.*
+import com.airbnb.lottie.LottieComposition
+import java.util.concurrent.TimeUnit
+
+/**
+ * Returns a mutable state representing the progress of an animation.
+ *
+ * Because the state is mutable, you can modify its value and the internal animation
+ * will continue animating from the value you set. The progress will snap to the value you
+ * set without changing the repeat count.
+ *
+ * There is also a suspending version of this that takes progress as a MutableState<Float>
+ * as a required second parameter.
+ *
+ * You do not have to use this to animate a Lottie composition. You may create your own animation
+ * and pass its progress to [LottieComposition].
+ *
+ * @param composition The composition to render. This should be retrieved with [lottieComposition].
+ * @param isPlaying Whether or not the animation is currently playing. Note that the internal
+ *                  animation may end due to reaching the target repeatCount. If that happens,
+ *                  the animation may stop even if this is still true. You may want to use
+ *                  onFinished to set isPlaying to false but in many cases, it won't matter.
+ * @param restartOnPlay If isPlaying switches from false to true, restartOnPlay determines whether
+ *                      the progress and repeatCount get reset.
+ * @param clipSpec A [LottieClipSpec] that specifies the bound the animation playback
+ *                 should be clipped to.
+ * @param speed The speed the animation should play at. Numbers larger than one will speed it up.
+ *              Numbers between 0 and 1 will slow it down. Numbers less than 0 will play it backwards.
+ * @param repeatCount The number of times the animation should repeat before stopping. It must be
+ *                    a positive number. [Integer.MAX_VALUE] can be used to repeat forever.
+ * @param onRepeat An optional callback to be notified every time the animation repeats. Return whether
+ *                 or not the animation should continue to repeat.
+ * @param onFinished An optional callback that is invoked when animation completes. Note that the isPlaying
+ *                   parameter you pass in may still be true. If you want to restart the animation, increase the
+ *                   repeatCount or change isPlaying to false and then true again.
+ */
+@Composable
+fun animateLottieComposition(
+    composition: LottieComposition?,
+    isPlaying: Boolean = true,
```

Comment:
Have you considered playMode vs. seekMode here, to avoid having isPlaying = true after animation is finished? :-)

Reply:
@doris4lt what would that look like, exactly?

Reply:
I seem to have misunderstood what `isPlaying` is for.  Sorry about the confusion... Upon reading the impl below, it is for pausing/resuming the animation.  

This really isn't a big deal. But I'd gravitate towards: `mode: PlayMode = Play`, where `enum class PlayMode { Play, Paused }` over isPlaying = true, since isPlaying implies the animation is ongoing and unfinished. Alternatively,  you could consider `isPaused`, `isPaused = false` doesn't imply the animation is finished or otherwise.